File: regexlib/javascript_test_file/regexlib_5839.js
/* 5839
 * &lt;(?:[^&quot;']+?|.+?(?:&quot;|').*?(?:&quot;|')?.*?)*?&gt;
 * EXPONENT
 * nums:3
 * EXPONENT AttackString:"&lt;"+"'"*2+"! _1! _1! _1! _1! _1!\n_SLQ_3"
 */
var REGEX = /&lt;(?:[^&quot;']+?|.+?(?:&quot;|').*?(?:&quot;|')?.*?)*?&gt;/
for(var i = 1; i <= 5000000; i++) {
    var time = Date.now();
    var attack_str = '&lt;' + '\''.repeat(i*1) + '! _1! _1! _1! _1! _1!\n_SLQ_3'
    REGEX.exec(attack_str)
    // REGEX.test(attack_str);
    // attack_str.search(REGEX)
    // attack_str.match(REGEX)
    var time_cost = Date.now() - time;
    console.log(i * 1 + ": " + time_cost+" ms")
}

File: regexlib/python_re2_test_file/regexlib_5839.py
# 5839
# &lt;(?:[^&quot;']+?|.+?(?:&quot;|').*?(?:&quot;|')?.*?)*?&gt;
# EXPONENT
# nums:5
# EXPONENT AttackString:"&lt;"+"'"*2+"! _1! _1! _1! _1! _1!\n_SLQ_3"

import re2 as re
from time import perf_counter

regex = """&lt;(?:[^&quot;']+?|.+?(?:&quot;|').*?(?:&quot;|')?.*?)*?&gt;"""
REGEX = re.compile(regex)
for i in range(0, 150000):
    ATTACK = "&lt;" + "\'" * i * 1 + "! _1! _1! _1! _1! _1!\n_SLQ_3"
    LEN = len(ATTACK)
    BEGIN = perf_counter()
    m = REGEX.search(ATTACK)
    # m = REGEX.match(ATTACK)
    DURATION = perf_counter() - BEGIN
    print(f"{i *1}: took {DURATION} seconds!")